src/Utilities/TupleFromTypelist.hpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <tuple>
+
+#include "Utilities/TMPL.hpp"
+
+namespace tuple_impl_detail {
+
+template <typename Elements>
+struct tuple_from_typelist_impl;
+
+template <typename... Elements>
+struct tuple_from_typelist_impl<tmpl::list<Elements...>> {
+  using type = std::tuple<Elements...>;
+};
+
+}  // namespace tuple_impl_detail
+
+/// \ingroup UtilitiesGroup
+/// A `std::tuple` from the types in a `tmpl::list`
+template <typename Elements>
+using tuple_from_typelist =
```

Comment:
This is `tmpl::wrap<Elements, std::tuple>`.

Reply:
Since the implementation is so simple, do you think it's worth adding the alias at all?

Reply:
I vote no, but since tagged_tuple_from_typelist is already in the code I'll see if anyone else wants to weigh in.  @kidder @nilsdeppe anything to add?

Reply:
I guess one (big) reason I can see having the type alias is that, even though I've used Brigand a lot, I was unaware of `tmpl::wrap`. If Brigand actually had documentation I would definitely agree that we don't need such a type alias. My worry is that we will end up getting several implementations of `tuple_from_typelist` wherever it's needed locally if we don't have some way of documenting that `tmpl::wrap<Elements, std::tuple>` exists. I can certainly see myself (before being involved in this conversation) having added such a helper struct somewhere. Just something I think we need to consider when deciding here :)

Reply:
Perhaps we could contribute documentation to Brigand instead of adding aliases to our code.. could be the same amount of time spent in a more useful way. I have been quite annoyed by Brigand’s lack of documentation before, and I haven’t seen its documentation improve at all.

Reply:
You're right.  This keeps coming up, and we should just fix it.  I'll get some sort of cheat sheet together for the dev guide of the stuff we've found useful in SpECTRE.  That should solve our immediate problems.  We can work on upstreaming after that.

Reply:
Turns out there's a `tmpl::as_tuple` that does exactly this.